mkpkg

```diff
@@ -158,11 +158,15 @@ class Functions:
                 continue
         return sorted(deps)
 
-    def mapbins(self, src, dst, *bins):
+    def mapfiles(self, src, dst, *bins, **kwargs):
+        suffix = self.pkg.vars['suffix'] if 'append_suffix' in kwargs and kwargs['append_suffix'] else ''
```

Comment:
This line is quite long. It could be split up as:
```python
    suffix = vars_suffix if (
        'append_suffix' in kwargs and kwargs['append_suffix']
        ) else ''
```
but I'm not sure if that makes it better.